lib/logstash/filters/translate.rb

```diff
@@ -81,7 +81,12 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
 
   # When using a dictionary file, this setting will indicate how frequently
   # (in seconds) logstash will check the dictionary file for updates.
-  config :refresh_interval, :validate => :number, :default => 300
+  config :refresh_interval, :validate => :number, :default => 5
+  # when using a dictionary file, if this is set to true, the dictionary's
+  # modification time will be checked before reloaded.  If the dictionary
+  # isn't modified, the refresh won't happen.  If you set this to false,
+  # increase the refresh_interval.
+  config :refresh_only_if_modified, :validate => :boolean, :default => true
```

Comment:
Does Logstash open this file in a mode (on windows) that works with shared access? My experience is that defaults are that if one program has a file open for writing, another program tries to open for reading and is denied. Are you sure this works with the way Logstash is opening the file today?